backend/tests/integration/mailing/test_views.py

```diff
@@ -76,3 +77,23 @@ def test_view_subscribe_to_list_with_missing_data(api_client, db, es_clear):
     assert expected_status_code == result_status_code
     assert expected_message == result_message
     assert expected_errors == result_errors
+
+
+def test_get_weekly_jobs_rss(app, api_client, redis, shared_datadir):
+    with patch.dict(app.config, {"WEEKLY_JOBS_EMAIL_REDIS_KEY": "weekly_jobs_email"}):
+        entry = {
+            "id": 12345,
+            "title": "New HEP positions opened last week",
+            "timestamp": 1568789887.583032,
+            "html_content": "HEP Jobs",
+        }
```

Comment:
this is not what gets put into redis: https://github.com/inspirehep/inspirehep/pull/451/files#diff-b0bc1d0b37b02da3a85dae03c090d755R66

Reply:
yep that's leftover format